[2023-03-21T00:03:33.005+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: project_masterdag.download_dataset_task scheduled__2023-03-19T23:56:30.039393+00:00 [queued]>
[2023-03-21T00:03:33.058+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: project_masterdag.download_dataset_task scheduled__2023-03-19T23:56:30.039393+00:00 [queued]>
[2023-03-21T00:03:33.059+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-03-21T00:03:33.059+0000] {taskinstance.py:1283} INFO - Starting attempt 2 of 3
[2023-03-21T00:03:33.060+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-03-21T00:03:33.141+0000] {taskinstance.py:1303} INFO - Executing <Task(PythonOperator): download_dataset_task> on 2023-03-19 23:56:30.039393+00:00
[2023-03-21T00:03:33.163+0000] {standard_task_runner.py:55} INFO - Started process 3995 to run task
[2023-03-21T00:03:33.188+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'project_masterdag', 'download_dataset_task', 'scheduled__2023-03-19T23:56:30.039393+00:00', '--job-id', '260', '--raw', '--subdir', 'DAGS_FOLDER/project_master_dags.py', '--cfg-path', '/tmp/tmpl9kc0hhi']
[2023-03-21T00:03:33.192+0000] {standard_task_runner.py:83} INFO - Job 260: Subtask download_dataset_task
[2023-03-21T00:03:33.233+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:250: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-03-21T00:03:33.482+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2023-03-21T00:03:33.717+0000] {task_command.py:388} INFO - Running <TaskInstance: project_masterdag.download_dataset_task scheduled__2023-03-19T23:56:30.039393+00:00 [running]> on host 796cac132c25
[2023-03-21T00:03:34.338+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=project_masterdag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-19T23:56:30.039393+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-19T23:56:30.039393+00:00
[2023-03-21T00:03:34.413+0000] {logging_mixin.py:137} INFO - Warning: Your Kaggle API key is readable by other users on this system! To fix this, you can run 'chmod 600 /home/***/.kaggle/kaggle.json'
[2023-03-21T00:03:34.450+0000] {logging_mixin.py:137} INFO - file path is /opt/***
[2023-03-21T00:03:34.450+0000] {project_master_dags.py:43} INFO - Downloading files...
[2023-03-21T00:06:05.389+0000] {project_master_dags.py:45} INFO - Downloading completed!
[2023-03-21T00:06:05.396+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-03-21T00:06:05.421+0000] {taskinstance.py:1326} INFO - Marking task as SUCCESS. dag_id=project_masterdag, task_id=download_dataset_task, execution_date=20230319T235630, start_date=20230321T000333, end_date=20230321T000605
[2023-03-21T00:06:05.500+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-03-21T00:06:05.555+0000] {taskinstance.py:2585} INFO - 1 downstream tasks scheduled from follow-on schedule check
